pkgs/top-level/python-packages.nix

```diff
@@ -27805,6 +27805,36 @@ in {
     };
   };
 
+  worldengine = buildPythonPackage rec {
```

Comment:
In this case we usually put them outside of `python-packages`, because we only need one version of it.